src/ui/app/index.tsx

```diff
@@ -39,7 +39,7 @@ const App = () => {
       <Route path={`/${pathPrefix}/login`} element={user && user.apiKey ? <Navigate to="/" replace /> : <LoginPage />} />
       <Route path={`/${pathPrefix}/account`} element={<RequireAuth user={user}><AccountPage user={user} /> </RequireAuth>} />
       <Route path={`/${pathPrefix}/workflow/:id`} element={<RequireAuth user={user}><WorkflowPage user={user} /> </RequireAuth>} />
-      <Route path={`/${pathPrefix}/workflow/:workflowId/result/:workflowDagResultId/function/:operatorId`} element={<RequireAuth user={user}><FunctionDetailsPage user={user} /> </RequireAuth>} />
+      <Route path={`/${pathPrefix}/workflow/:workflowId/result/:workflowDagResultId/operator/:operatorId`} element={<RequireAuth user={user}><FunctionDetailsPage user={user} /> </RequireAuth>} />
```

Comment:
I had named the path to .../function/:id to preemptively reduce confusion because operator is also a generic term we've been using to refer to metric operators, check operators... but this route is specifically for the function operator details pages.

Would love to know your thoughts on this

 to function because we also called metrics metricOperator so I wanted to distinguish the function/execution(?) operators links from being a generic operator link.

Reply:
+ if you make this change I believe we need to change src/ui/common/src/components/pages/**function**/ to src/ui/common/src/components/pages/**operator**/

Reply:
I think we should go with operator instead of function since that's the terminology used in many places in our code base.

We should also update the "View Function Details" button to say "View Operator Details"

Creating a linear task to address this work as a follow up:

Reply:
https://linear.app/aqueducthq/issue/ENG-1769/rename-function-details-to-operator-details